src/Umbraco.Infrastructure/BackOffice/BackOfficeUserManager.cs

```diff
@@ -181,14 +202,16 @@ public async Task<IdentityResult> ChangePasswordWithResetAsync(int userId, strin
             if (user == null) throw new InvalidOperationException("Could not find user");
 
             var result = await base.ResetPasswordAsync(user, token, newPassword);
-            if (result.Succeeded) RaisePasswordChangedEvent(userId);
+            if (result.Succeeded)
+                RaisePasswordChangedEvent(null, userId); // TODO: How can we get the current user? we have not HttpContext (netstandard), we can make our own IPrincipalAccessor?
```

Comment:
I will create a task for this. There's several options and I'm unsure if `IPrincipalAccessor` is the best option since this will be the only place that it is used (at least currently). The other option(s) which all have benefits are:

* We interface BackOfficeUserManager and only declare what we use, then this is implemented at the aspnetcore level using aspnetidentity, then we don't have to worry about this. Similarly i have TODOs in my current branch to do this for the BackOfficeSignInManager as i think it would be better to have our own abstractions for what we use which is implemented by aspnetidentity but at the core it doesn't matter.
* We abstract away the BackOfficeUserManager event manager which is injected and implemented at the aspnetcore level which does the event management, then we don't have to worry about this problem.

Of course we could do both which might make sense. I'll create a different task for this.